resolwe/toolkit/tests/test_dirs.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=missing-docstring
+import os
+
+from resolwe.flow.models import Data
+from resolwe.test import ProcessTestCase, tag_process, with_docker_executor
+
+
+class DirsProcessTestCase(ProcessTestCase):
+
+    def setUp(self):
+        super(DirsProcessTestCase, self).setUp()
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'files'))
+
+    @with_docker_executor
+    @tag_process('upload-dir')
+    def test_upload_dir(self):
+        # Complete tree structure of 'compressed dir.tar.gz'
+        structure = {
+            'file binary': None,
+            'text file.txt': None,
+            'root dir': {
+                'file binary': None,
+                'text file.txt': None,
+                'nested dir.ext': {
+                    'empty dir': {
+                    },
+                },
+            },
+        }
+
+        partial_structure = {
+            'root dir': {
+                'file binary': None,
+            },
+        }
+
+        sym_link = {
+            'text file.txt': None,
+            'sym link': None,
+        }
+        upload_dir = self.run_process('upload-dir', {'src': 'compressed dir.tar.gz'})
+        self.assertDirExists(upload_dir, 'dir')
+        # https://github.com/PyCQA/pylint/issues/1653
+        self.assertRaisesRegex(  # pylint: disable=deprecated-method
```

Comment:
This is a test for `ProcessTestCase.assertDirExists`, not for the process.

Reply:
Removed.